### PR TITLE
added userLeaderBoard and Mdo LeaderBoard api's

### PR DESCRIPTION
**Pull request by shankaragoudab**

### CODE REVIEW

File: src/utils/whitelistApis.ts [hunk 1]
```diff
@@ -4645,6 +4659,8 @@ export const API_LIST = {
             '/proxies/v8/mentoring/v1/entity-type/read',
             '/proxies/v8/mentoring/v1/entity/list',
             '/proxies/v8/data/v2/system/settings/get/:id',
-            '/proxies/v8/mentoring/v1/sessions/update'
+            '/proxies/v8/mentoring/v1/sessions/update',
+            '/proxies/v8/halloffame/v1/mdoleaderboard',
+            '/proxies/v8/halloffame/v1/userleaderboard/:id'
```
Comment (Sahil-tarento) [inline src/utils/whitelistApis.ts]: Please add '/proxies/v8/halloffame/v1/userleaderboard/:id',
comma at the last

File: src/proxies_v8/proxies_v8.ts [hunk 2]
```diff
@@ -985,6 +985,10 @@ proxiesV8.use('/competencySubTheme/*',
   proxyCreatorSunbird(express.Router(), `${CONSTANTS.KONG_API_BASE}`)
 )
 
+proxiesV8.use('/halloffame/*',
```
Comment (Sahil-tarento) [inline src/proxies_v8/proxies_v8.ts:993]: Please check I think halloffame is already there

**Review by Sahil-tarento — CHANGES_REQUESTED** [review]

File: src/utils/whitelistApis.ts (hunk 1 above)
Comment (shankaragoudab) [reply src/utils/whitelistApis.ts]: added

File: src/proxies_v8/proxies_v8.ts (hunk 2 above)
Comment (shankaragoudab) [reply src/proxies_v8/proxies_v8.ts:993]: yes halloffame api's are there but prefix was not there

**Pull request merged**